Final_Project/Main.cpp

```diff
@@ -194,7 +202,18 @@ LRESULT CALLBACK WndProc(HWND hwnd, UINT Message,
 			char ctrlFrame[4] = {};
 			sendFrame(ctrlFrame, NULL, ENQ);
 			ENQ_FLAG = true;
-			
+
+
+			char dataFrame[12] = {}; //for test; to be removed
```

Comment:
These examples are here for testing. Once the pull request is approved they will get removed.